src/nvim/tui/term_input.inl

```diff
@@ -269,7 +269,11 @@ static TermInput *term_input_new(void)
     flags |= TERMKEY_FLAG_RAW;
   }
 
-  rv->tk = termkey_new_abstract(os_getenv("TERM"), flags);
+  const char *term = os_getenv("TERM");
+  if (!term) {
+    term = "";  // termkey_new_abstract assumes non-null (#2745)
+  }
+  rv->tk = termkey_new_abstract(term, flags);
```

Comment:
I'll look at this more tomorrow, but I could swear my reading of this function indicated it should return `NULL` when `term` is empty, and yet based on my testing, it's not. Either way, we should have a strategy for dealing with that. Premature exit?